### PR TITLE
Update constraints in order to allow some "symfony/*:^5.2" deps

### PR DESCRIPTION
**Pull request by phansys**
## Subject
Update constraints in order to allow some "symfony/*:^5.2" deps.

I am targeting this branch, because these changes respect BC.

## Changelog

```markdown
### Added
- Support for symfony/config:^5.2;
- Support for symfony/dependency-injection:^5.2;
- Support for symfony/http-foundation:^5.2.
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,13 +32,13 @@
...
         "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4",
+        "symfony/dependency-injection": "^4.4 || ^5.2",
         "symfony/doctrine-bridge": "^4.4",
         "symfony/form": "^4.4",
         "symfony/framework-bundle": "^4.4",
-        "symfony/http-foundation": "^4.4",
+        "symfony/http-foundation": "^4.4 || ^5.2",
         "symfony/http-kernel": "^4.4",
         "symfony/options-resolver": "^4.4 || ^5.1",
```
Comment (VincentLanglet) [inline composer.json:43]: Should you update the 5.1 ?

Comment (OskarStark) [reply composer.json:43]: If 5.1 works, why?

Comment (VincentLanglet) [reply composer.json:43]: So why using `^5.2` for others ?

We could even use `^5.0` every time.
I thought our policy was to allow only supported version. https://symfony.com/releases

Comment (OskarStark) [reply composer.json:43]: Sure but this Pr is for supporting new versions, why should we raise the bar for a version which we already support?

Comment (phansys) [reply composer.json:43]: If you agree, I can use 5.1 or 5.2 based on the decision we take.
Please, let me know what your suggestions are in order to get this PR merged.

Comment (OskarStark) [reply composer.json:43]: I don't want to be too picky, I am fine with a bump to 5.2

Comment (VincentLanglet) [reply composer.json:43]: I dont really have preference about the one chosen but Id like to have consistency :)

Comment (phansys) [reply composer.json]: Thank you!

**Review by dmaicher — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks !

**Pull request merged**